.circleci/config.yml

```diff
@@ -320,26 +294,6 @@ jobs:
             - notify-failures-on-develop:
                 mentions: "@proofs-team"
 
-  cannon-build-test-vectors:
```

Comment:
This is just getting removed, but not replaced? I assume it's only applicable to 32bit targets?

Reply:
Correct. The 64bit version wrote tests in a much simpler way so they don't need to be built separately like this (deleting the tests was a big part of the 50k LoC that got deleted).